docs/monitoring-usage.rst

```diff
@@ -154,6 +154,85 @@ before you call
     projects.metricDescriptors
 
 
+Groups
+------
+
+A group is a dynamic collection of *monitored resources* whose membership is
```

Comment:
Small nit, a group isn't a who, so "whose" is incorrect. Though I probably would've written "whose" as well.